content/06.discussion.md

```diff
@@ -54,6 +54,34 @@ In assessing the upper bound on the predictive performance of a deep learning mo
 Study-level variability limits classification performance and can lead to underestimating prediction error if the generalization error is estimated by splitting a single dataset.
 Analyses can incorporate data from multiple labs and experiments to capture between-study variation within the prediction model mitigating some of these issues.
 
+### Uncertainty quantification
+
+Deep learning based solutions for biomedical applications require reliability guarantees and a quantification of uncertainties. Due to biological variability and precision limits of equipment, biomedical data do not consist of precise measurements but of estimates with noise. Hence, it is crucial to obtain uncertainty measures that capture how noise in input values propagate through deep neural networks. Such measures can be used for reliability assessment of automated decisions in clinical and public health applications, and for guarding against model vulnerabilities in the face of rare or adversarial cases [@tag:ghahramani_protect]. Moreover, in fundamental biological research, measures of uncertainty help researchers distinguish between true regularities in the data and patterns that are false or merely anecdotal. There are two main uncertainties that one can calculate: epistemic uncertainty, which describes uncertainty about the model and is caused by insufficient training data or by a difference in the training set and testing set distributions (this uncertainty vanishes in the limit of infinite data), and aleatoric uncertainty, which describes uncertainty inherent in the observations and is due to noisy or missing data (this uncertainty vanishes with the ability to observe all independent variables with infinite precision) [@tag:uncertainty_types]. Aleatoric uncertainty can be represented well by designing an appropriate loss function. In the case of data-dependent aleatoric uncertainty, one can train the model to increase its uncertainty when it is incorrect due to noisy or missing data, and in the case of task-depedent aleatoric uncertainty, one can optimize for the best uncertainty parameter for each task [@tag:uncertainty_multi_task]. Meanwhile, there are various methods for modeling epistemic uncertainty estimates.
```

Comment:
Can you do one sentence per line? It will make it easier to comment on specific portions. I have reviewed the above paragraph and I have only one major suggestion. You are about to drop into the subsections. Could you discuss which subsections relate to each type of uncertainty?

Reply:
All subsections discuss modeling epistemic uncertainty. I added a clarification.